### PR TITLE
Adds a setup.py file

### PR DESCRIPTION
**Pull request by vmesel**
As mentioned in #83, I've added a sample setup.py file.

### CODE REVIEW

**Comment by imnotbrandon** [comment]
Thanks for the contribution!
I guess the next logical step would be for us to package it up and submit the tool to pypi.

**Comment by imnotbrandon** [comment]
Could you also update the readme with instructions on how to install the tool please?

**Comment by vmesel** [comment]
@imnotbrandon yes, I can! going to do this in a while!

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+setup(
+    name='dnsReaper',
+    version='1.2.0',
+    packages=['providers', 'signatures', 'signatures.checks', 'signatures.templates'],
+    url='',
+    license='GNU Affero',
+    author='Punk Security',
+    author_email='',
+    description='subdomain takeover tool for attackers, bug bounty hunters and the blue team!'
```
Comment (Dalyopbala) [inline setup.py:11]: 0xa0f98321f02e90e13d04b08c1b9f3252dde15fd3

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+from distutils.core import setup
+
+setup(
+    name='dnsReaper',
+    version='1.2.0',
```
Comment (Dalyopbala) [inline setup.py:5]: Dope

**Comment by vmesel** [comment]
@imnotbrandon take a look pls!

**Review by imnotbrandon — DISMISSED** [review]

**Comment by vmesel** [comment]
@imnotbrandon can you re-verify the code, please? Now I think the signature for the commit was added

**Comment by imnotbrandon** [comment]
Hi,
Thanks - could you update the version to 1.5.0 please? We'll maintain the version number from then on!

Also the commits are still not showing up as verified on my end. I won't be able to merge the PR until the commits are signed.

**Pull request closed without merging**